scripts/archive/deploy-from-staging.sh

```
@@ -69,14 +69,14 @@ echo -e "\n${YELLOW}Step 3: Preparing staging images for production${NC}"
 echo "Tagging staging images as production..."
 
 # Tag staging images as production
-docker tag cell-segmentation-hub-frontend:latest cell-segmentation-hub-frontend:production-$(date +%Y%m%d-%H%M%S)
-docker tag cell-segmentation-hub-backend:latest cell-segmentation-hub-backend:production-$(date +%Y%m%d-%H%M%S)
-docker tag cell-segmentation-hub-ml-service:latest cell-segmentation-hub-ml-service:production-$(date +%Y%m%d-%H%M%S)
+docker tag spheroseg-app-frontend:latest spheroseg-app-frontend:production-$(date +%Y%m%d-%H%M%S)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)